time.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"golang.org/x/sys/unix"
+)
+
+const agentStartedEvent = "agent_started"
+
+//Event that will be reported for time tracing
+type eventTime struct {
+	Event string
+	Time  unix.Timespec
```

Comment:
The actual timestamp itself should be in https://en.wikipedia.org/wiki/ISO_8601 format, or the RFC 3339 variant used by https://golang.org/pkg/time/. Looks like logrus uses that by default.